### PR TITLE
feat(card/arrow): link card with arrow icon

### PR DESCRIPTION
**Pull request by AgonisticKatai**
image

(**In order to be able to launch SUI-Studio locally, I had to fix some linter bugs in other components.**)

### CODE REVIEW

File: components/card/arrow/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/* sass-lint:disable no-important*/
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
```
Comment (JoanClaret) [inline components/card/arrow/src/index.scss]: Not necessary, i think you can delete it.

File: components/card/arrow/src/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+   */
+  icon: PropTypes.func,
+  /**
+   * Factory for the component that will hold the card link.
+   */
+  linkFactory: PropTypes.func,
+  /**
+   * URL for the link that wraps the whole card.
+   */
+  link: PropTypes.string
```
Comment (JoanClaret) [inline components/card/arrow/src/index.js]: `.isRequired`?

File: components/card/arrow/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+$ npm install @schibstedspain/sui-card-arrow --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import CardArrow from '@schibstedspain/sui-card-arrow'
+
+return (<CardArrow />)
```
Comment (JoanClaret) [inline components/card/arrow/README.md]: maybe we could explain how to use it with props

```
const link = 'https://www.lorem.com/'

const media = {
  alt: 'Lorem image',
  src: 'https://www.lorem.com/ipsum.jpg'
}

const text = {
  title: 'Lorem',
  description: 'Lorem ipsum dolor sit amet'
}

<CardArrow media={media} text={text} link={link}/>
```

File: components/card/arrow/src/index.scss (hunk 1 above)
Comment (AgonisticKatai) [reply components/card/arrow/src/index.scss]: In order to use the @include sui-card, I need @import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';

Comment (JoanClaret) [reply components/card/arrow/src/index.scss]: are you sure? 
this mixin isn't located in the `settings-compat` https://github.com/SUI-Components/sui-theme/blob/master/src/components/_card.scss#L6

Comment (AgonisticKatai) [reply components/card/arrow/src/index.scss]: Some of the variables used by sui-card are defined in settings-compat-v7. 🤷‍♂️

Comment (JoanClaret) [reply components/card/arrow/src/index.scss]: ok, i see. This means that non deprecated components are using a deprecated config file.

Same happens with buttons
https://github.com/SUI-Components/sui-theme/blob/master/src/components/_button.scss#L1
Which is using a deprecated file
https://github.com/SUI-Components/sui-theme/blob/master/src/settings-compat-v7/_components.scss#L81

and some other components.

is this something that we expect @jcger ?
should we move this vars outside the `compat` folder?

anyways, this is not related with this PR, IMHO we can manage this in a further PR.

Comment (jcger) [reply components/card/arrow/src/index.scss]: Yep, it's tech debt we will face once we end developing all of our atoms and molecules

File: components/form/pta/src/index.js [hunk 4]
```diff
@@ -16,11 +16,17 @@ class FormPta extends Component {
   constructor(props) {
     super(props)
 
-    const {formUrl: BASE_URL, onSubmit, onError, eventListeners, ...settings} = this.props
+    const {
```
Comment (jcger) [inline components/form/pta/src/index.js]: this changes shouldn't be part of this PR, right?

Comment (jcger) [reply components/form/pta/src/index.js]: In this case it shouldn't be a big deal but you are changing form/pta stuff under the card/arrow scope so no release changes will be found

Comment (AgonisticKatai) [reply components/form/pta/src/index.js]: ok, being very controlled changes, and not needing to make a new release of the form/pta, I upload it and in a future modification of the form/pta we solve it.
Thank you very much!
